### PR TITLE
[incubator/metricbeat] metricbeat chart

### PR DESCRIPTION
**Pull request by aba182**
Signed-off-by: Adam Wilhelm <ajwilhel@gmail.com>

**What this PR does / why we need it**: adds a metricbeat chart to community charts

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: New chart

**Special notes for your reviewer**:

### CODE REVIEW

File: incubator/metricbeat/templates/daemonset-map.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
```
Comment (cpanato) [inline incubator/metricbeat/templates/daemonset-map.yaml]: you can remove this line

Comment (aba182) [reply incubator/metricbeat/templates/daemonset-map.yaml]: Would you like me to remove the `---` header from all yaml files?

Comment (cpanato) [reply incubator/metricbeat/templates/daemonset-map.yaml]: why do you need that?

Comment (cpanato) [reply incubator/metricbeat/templates/daemonset-map.yaml]: just to make similar to other charts

Comment (aba182) [reply incubator/metricbeat/templates/daemonset-map.yaml]: Thanks, I have removed. I had just converted this chart from a single config file so they were present from my source and hung around :)

**Comment by aba182** [comment]
PR: https://github.com/helm/charts/pull/7347 is doing the same thing and was merged to stable. Closing my PR and testing the other version now.

**Pull request closed without merging**